.changeset/lovely-snakes-pump.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
We'll want to have a changelog entry for this change, so please add one.

Since this is a small change I'd also suggest to make this a patch release, not a minor release.